.github/workflows/release.yml

```diff
@@ -212,13 +213,13 @@ jobs:
 
       - name: compress musl artifact
         run: |
-          tar --strip-components 1 -czf resolc-static-linux.tar.gz ./resolc-out/resolc-static-linux
+          tar -czpf $(pwd)/resolc-static-linux.tar.gz -C ./resolc-out resolc-static-linux
```

Comment:
Is `-p` to preserve the permissions or should it be `--xattrs`?

Reply:
AFAIK permissions are preserved by default. `lipo` probably doesn't set the `x` bit?